core/project_settings.cpp

```diff
@@ -44,6 +44,7 @@
 #include <zlib.h>
 
 ProjectSettings *ProjectSettings::singleton = NULL;
+const int ProjectSettings::CONFIG_VERSION = 4;
```

Comment:
It wouldn't compile otherwise.

I believe I needed to do this because of templates. Since there would have been multiple instantiation of the same variable from the templates copying the variable everywhere....

Reply:
That doesn't make much sense to me, it must have been unrelated to this PR.